src/main/java/TaskList.java

```diff
@@ -0,0 +1,71 @@
+import java.util.ArrayList;
+
+public class TaskList {
+    ArrayList<Task> ls = new ArrayList<>();
+
+    public TaskList(ArrayList<Task> list) {
+        ls.addAll(list);
+    }
+
+    /**
+     * Adds param task into this.ls
+     * @param task
+     */
+    void add(Task task) {
+        ls.add(task);
+    }
+
+    /**
+     * Returns size of this.ls
+     * @return int
+     */
+    int size() {
+        return ls.size();
+    }
+
+    /**
+     * Marks the task at index in this.ls as done
+     * @param index
+     */
+    void done(int index) {
```

Comment:
Maybe a more meaningful name "completeTask/markAsDone" would be more descriptive?